.github/workflows/test.yml

```diff
@@ -14,12 +14,14 @@ jobs:
     steps:
       - name: Smoke test TUF-on-CI repository with a TUF client
         uses: theupdateframework/tuf-on-ci/actions/test-repository@4ae5fdf2b18c6256787df167fce7a0e2d7e7a40e # v0.5.0
+        with:
+          metadata_url: https://sigstore.github.io/root-signing-staging/
 
   update-issue:
     runs-on: ubuntu-latest
     needs: [smoke-test]
     # During workflow_call, caller updates issue
-    if: always() && !cancelled() && github.workflow == 'test.yml'
+    if: always() && !cancelled() && github.workflow == 'TUF-on-CI repository tests'
```

Comment:
Oh, this is a bummer we can't use the filename :(

Reply:
I'm still kind of expecting there is a better way... but I could not think of one